python_scripts/02_numerical_pipeline_scaling.py

```diff
@@ -192,7 +192,23 @@
 data_train_scaled.describe()
 
 # %% [markdown]
-# We can easily combine these sequential operations with a scikit-learn
+# We can use a jointplot to visualize the histograms and scatterplot of any
+# pair of numerical features at the same time. We can observe that `StandardScaler`
+# does not change the structure of the data itself but the axes
+# get shifted and scaled.
+
+# %%
+import matplotlib.pyplot  as plt
+import seaborn as sns
+
+sns.jointplot(data=data_train[:300], x="age", y="hours-per-week", marginal_kws=dict(bins=15))
+plt.suptitle("Jointplot of 'age' vs 'hours-per-week' \nbefore StandardScaler", y = 1.1)
+
+sns.jointplot(data=data_train_scaled[:300], x="age", y="hours-per-week", marginal_kws=dict(bins=15))
+_ = plt.suptitle("Jointplot of 'age' vs 'hours-per-week' \nafter StandardScaler", y = 1.1)
```

Comment:
PEP8 no space around equal space for parameter `suptitle(..., y = 1.1)` -> `suptitle(..., y=1.1)`